private/buf/bufgen/generator.go

```diff
@@ -53,12 +54,13 @@ func newGenerator(
 	logger *zap.Logger,
 	storageosProvider storageos.Provider,
 	runner command.Runner,
+	wasmPluginExecutor bufwasm.PluginExecutor,
 	clientConfig *connectclient.Config,
 ) *generator {
 	return &generator{
 		logger:                logger,
 		storageosProvider:     storageosProvider,
-		appprotoexecGenerator: bufpluginexec.NewGenerator(logger, storageosProvider, runner),
+		appprotoexecGenerator: bufpluginexec.NewGenerator(logger, storageosProvider, runner, wasmPluginExecutor),
```

Comment:
Not from this PR but would be nice if we could quickly cleanup: this field should be renamed pluginexecGenerator